react/ProductList.tsx

```diff
@@ -54,7 +54,7 @@ const ProductList: StorefrontFunctionComponent<Props> = ({
     ))
 
   return (
-    <Fragment>
+    <div>
```

Comment:
Add an `id` to this `div` to avoid this kind of mistake in the future, maybe `unavailable-items-wrapper`.